pyodide-build/pyodide_build/buildpkg.py

```diff
@@ -734,36 +794,62 @@ def make_parser(parser: argparse.ArgumentParser):
     return parser
 
 
+def parse_continue_arg(continue_from: Optional[str]) -> Dict[str, Any]:
+    assert (
+        continue_from is None
+        or continue_from == "capture"
+        or re.fullmatch("replay(:[0-9]*)?", continue_from)
+    )
```

Comment:
Similarly, we need explicit exceptions, otherwise, this will be undebugable when not run in a debugger.